homework/test.js

```diff
@@ -0,0 +1,56 @@
+import { expect } from 'chai';
+import {  ShopError, Aliments, Furniture, Customer, addProductTest, fundsTest, emptyCartTest, paymentTest } from './homework.js';
+
+describe('AddAliment', () => {
+    it('Valid Aliment', () => {
+        var futureDate = new Date(); 
+        futureDate.setDate(new Date() + 1);
+
+        var testAliment = new Aliments('testAliment', 20, futureDate, 100);
+
+        expect(addProductTest(testAliment)).to.be.eq(true);
+    });
+
+    it('Expired Aliment', () => {
+        var pastDate = new Date();
+        pastDate.setDate(pastDate.getDate() - 1);
+
+        var testAliment = new Aliments('testAliment', 20, pastDate, 100);
+        
+        expect(() => addProductTest(testAliment)).to.throw(ShopError, 'Customer can not add an expired aliment to their shopping cart.');
```

Comment:
You should import the error class from the tested source code and make use of its `ERROR_MESSAGE` static property to access the error message instead of hardcoding it.